frontend/packages/operator-lifecycle-manager/src/components/operand/const.ts

```diff
@@ -37,7 +37,7 @@ export const REGEXP_FIELD_DEPENDENCY_CAPABILITY = _.escapeRegExp(SpecCapability.
 export const REGEXP_SELECT_CAPABILITY = _.escapeRegExp(SpecCapability.select);
 
 export const REGEXP_K8S_RESOURCE_SUFFIX = new RegExp(
-  `^${REGEXP_K8S_RESOURCE_CAPABILITY}(?:core~v1~)?(.*)$`,
+  `^${REGEXP_K8S_RESOURCE_CAPABILITY}(?:core[:~]v1[:~])?(.*)$`,
```

Comment:
No, because `(?:core[:~]v1[:~])?` is an optional, non-capturing group. So if it is a core/v1 kind, and the whole gvk is provided, this will only capture the kind, but all other' will capture the entire gvk string.